va/validation-authority.go

```diff
@@ -167,7 +167,7 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 	va.log.AuditErr(fmt.Errorf("Attempting to validate %s for %s", challenge.Type, url))
 	httpRequest, err := http.NewRequest("GET", url.String(), nil)
 	if err != nil {
-		va.log.Debug(fmt.Sprintf("%s [%s] HTTP failure: %s", challenge.Type, identifier, err))
+		va.log.Err(fmt.Sprintf("Failed to parse URL '%s'. ErrType=%#v ErrStr=%s", identifier, err, err))
```

Comment:
Can you use `err=[%#v] and errStr=[%s]` to be a little more consistent with the CA code that logs errors?